ouroboros-consensus-cardano/tools/db-analyser/Analysis.hs

```diff
@@ -215,7 +211,7 @@ processAllChainDB chainDB rr callback = do
       Origin -> return ()
       At tip -> do
         Right itr <- ChainDB.stream chainDB rr GetBlock
-          (StreamFromExclusive genesisPoint)
+          (StreamFromExclusive GenesisPoint)
```

Comment:
Is this change related?

Reply:
`Ouroboros.Consensus.Block` re-exports most from `Ouroboros.Network.Block`, but not `genesisPoint`, only the `GenesisPoint` pattern synonym.